Patch-Source: https://github.com/SSSD/sssd/pull/7989
---
From 5793a46e12c7673456b6dfd465a8e49757dc0bc1 Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Fri, 6 Jun 2025 08:03:06 +0200
Subject: [PATCH] sss_prctl: avoid redefinition of prctl_mm_map

prctl_mm_map is provided by linux's prctl.h and libc's prctl.h.
libc's headers should be preferred.
---
 src/util/sss_prctl.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/util/sss_prctl.c b/src/util/sss_prctl.c
index c7561673b51..aa29bcdb198 100644
--- a/src/util/sss_prctl.c
+++ b/src/util/sss_prctl.c
@@ -21,7 +21,6 @@
 #include "util/sss_prctl.h"
 
 #ifdef HAVE_PRCTL
-#include <linux/prctl.h>
 #include <sys/prctl.h>
 
 int sss_prctl_set_dumpable(int dumpable)
